### PR TITLE
Sink me! Thee cargo is nix! 

### PR DESCRIPTION
**Pull request by workflow**
The nixens be strong in this one.

`nix-build -A package`

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        "warn"
+      ];
+    };
+    environment = {
+      "registry+https://github.com/rust-lang/crates.io-index".openssl-sys."*".OPENSSL_DIR = openssl pkgs;
+    };
+
+    buildInputs = {
+      unknown.rmob."*" = with pkgs; [ libiconv ] ++ macosFrameworks;
+      "registry+https://github.com/rust-lang/crates.io-index".cargo."*" = with pkgs; [ libiconv ] ++ macosFrameworks;
```
Comment (ebkalderon) [inline default.nix]: I think this should be okay to remove, since `rmob` doesn't depend on `cargo`.

File: default.nix [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  rustc = rustChannel.rust.override {
+    targets = [
+      (pkgs.rustBuilder.rustLib.realHostTriple pkgs.stdenv.targetPlatform)
+      "aarch64-unknown-linux-gnu"
+    ];
+  };
+
+  # source filter
+  srcFilter = {src, name, type}:
+    (type == "directory" && name == "${toString src}/overlay" -> false) &&
```
Comment (ebkalderon) [inline default.nix]: This is okay to remove since we don't have an `overlay` directory at the root of the repo.

File: default.nix (hunk 1 above)
Comment (workflow) [reply default.nix]: Sail ho! Good advice

File: default.nix (hunk 2 above)
Comment (workflow) [reply default.nix]: Fool! Good advice

**Pull request merged**